packages/office-ui-fabric-react/src/components/TeachingBubble/docs/new/TeachingBubbleBestPractices.md

```diff
@@ -0,0 +1,15 @@
+### Layout
+
+- TeachingBubbles can be used in sequence to walk people through complex, multistep interactions only. And only show one TeachingBubble at a time.
+  - A maximum of no more than 3 sequenced TeachingBubbles should be used in a single experience.
+  - Sequenced TeachingBubbles should have “x of y” text to indicate progress through the sequence. For example, the first TeachingBubble in a sequence might be “1 of 3”.)
+  - The action button on a sequenced TeachingBubble should read “Next” and the final sequenced TeachingBubble should have an action button with text “Got it”, that closes the experience.
```

Comment:
I'm not sure we need to worry about the localized text issue here since this is guidance for consumers of the control (not strings we'd put in a Fabric control), and if their apps/sites are localized, they should have some localization pipeline set up and know to run all strings through it.